RELEASE.txt

```diff
@@ -101,14 +113,24 @@ the corresponding release branch.
   - Open that URL and check for the wheels as
     ``scikit_image-<versiontag>-<platforminfo>.whl``
 
-- Update the docs:
+**Do not tag until the builds are all green.**
+
+- Add the version number as a tag in git::
+
+   git tag -s -m <github_release_message> [-u <key-id>] v<major>.<minor>.0
+
+  (If you do not have a GPG key, follow the tutorial to set it up:
+   https://help.github.com/articles/signing-commits-with-gpg/)
+
+- Push the new tag to GitHub::
+
+   git push upstream v<major>.<minor>.0
+
+  (where ``upstream`` is the name of the
+   ``github.com:scikit-image/scikit-image`` repository.)
 
-  - Edit ``doc/source/_static/docversions.js`` and commit
-  - Build a clean version of the docs. In the root directory, run
-    ``python setup.py install``, then ``make cleandoc; make html``.
-  - In the ``doc/`` directory, build using ``make gh-pages``.
-  - Update the symlink to ``stable``.
-  - Push upstream: ``git push origin gh-pages`` in ``doc/gh-pages``.
+- Build the package wheels again, but change BUILD_COMMIT in both
+  Travis-CI and AppVeyor to the release version, e.g., ``v0.16.0``.
```

Comment:
some notes should be added below ensuring that you are using a clean up to date environment for sdist generation. sdists ship generated c code and should be done with the latest cython version to support the most python versions. (old versions of cython will not support say 3.8)

Reply:
We should probably stop shipping generated C code, it's easy enough to install Cython nowadays.

Reply:
(We had a great mechanism before, which was to generate .md5 hash files upon Cythonization.  We would *not* ship these files, so whenever a user who had Cython installed tried to compile skimage, Cython would be called.  Otherwise, it would fall back on the pre-generated files.  When we switched to using Cython's internal mechanism for determining when to recompile, we lost that advantage.)

Reply:
Right. I can make that PR.

Reply:
Thanks!